src/App.js

```diff
@@ -1,25 +1,69 @@
 import React from 'react';
-
+import { Clock } from './components/Clock';
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+class App extends React.Component {
+  state = {
+    clockName: Math.floor(Math.random() * 1000),
+    time: new Date(),
+    isClockVisible: true,
+  }
+
+  componentDidMount() {
+    setInterval(() => {
+      this.setState({ time: new Date() });
+    }, 1000);
+  }
```

Comment:
clearInterval in ComponentWillUnmount
https://reactjs.org/docs/state-and-lifecycle.html#adding-lifecycle-methods-to-a-class